### ISSUE TITLE
Use higher-resolution brave_file.* ICO files

### ISSUE THREAD

**Issue by jonathansampson** [brave/brave-browser#41901]
### Description

A few years ago we added custom icon support (https://github.com/brave/brave-core/commit/63b23efb43741287dcebbfa5e3613132460299ff) for PDFs and SVGs. This change was exclusively for Windows, and long preceded a recent commit (https://github.com/chromium/chromium/commit/a6d8f06ca86341e008497192f7c0c90cf73113d8) by the Chromium team to land similar functionality.

On Windows, ICO files should contain multiple versions/sizes—see [/chromium/…/README.md#ico-images](https://github.com/chromium/chromium/blob/main/chrome/app/theme/README.md#ico-images) for specifics. The ICO files we added (i.e., brave_file.ico, brave_file_beta.ico, etc.) all consist of a single 32x32 image (the Brave logo appears even smaller, due to visual padding around the icon).

We should consider the recent Chromium update. But in the meantime, we could also simply create a new set of ICO files to give Windows users a better experience.

### Steps to reproduce

1. Configure Brave to be the default handler for SVG/PDF types

### Actual result

Icons for files are small and blurry

![Image](https://github.com/user-attachments/assets/854fb454-962a-412f-b2e3-d4b545281e9e)

### Expected result

Icons should be adaptive to the display size (up to extra large), similar to the application icon.

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

1.71.118 Chromium: 130.0.6723.70 (Official Build) (64-bit) 

### Channel information

- [x] release (stable)
- [x] beta
- [x] nightly

### Reproducibility

- [x] with Brave Shields disabled
- [x] with Brave Rewards disabled
- [ ] in the latest version of Chrome

**Comment by aguscruiz** [comment]
@jonathansampson let me know if you're good with the assets I provided earlier today, or if you need anything else :D

**Comment by jonathansampson** [comment]
> [@jonathansampson](https://github.com/jonathansampson) let me know if you're good with the assets I provided earlier today. Do you need the one for .svg too or are we good with the pdf one ?

They should work great! I took some time this morning to build https://sampson.codes/brave/ico-viewer/ as a means of verifying future ICOs. Glad I did, because I nearly proceeded with an ICO that had too many 8-bit images within 🙂

Here are the results for the two ICOs I constructed from your source images:

![Image](https://github.com/user-attachments/assets/f8ba3b91-07cb-4277-9281-ee67853892f6)
![Image](https://github.com/user-attachments/assets/289f31c0-f501-42e8-8116-1283c707525b)

**Comment by aguscruiz** [comment]
Awesome! 🤩

**Comment by aguscruiz** [comment]
@jonathansampson is this ok to be closed?
